packaging/pkg_helpers.bash

```diff
@@ -180,13 +180,10 @@ setup_wheel_python() {
   if [[ "$(uname)" == Darwin || "$OSTYPE" == "msys" ]]; then
     eval "$(conda shell.bash hook)"
     conda env remove -n "env$PYTHON_VERSION" || true
-    if [[ "$PYTHON_VERSION" == 3.9 ]]; then
-      export CONDA_CHANNEL_FLAGS="${CONDA_CHANNEL_FLAGS} -c=conda-forge"
-    fi
     conda create ${CONDA_CHANNEL_FLAGS} -yn "env$PYTHON_VERSION" python="$PYTHON_VERSION"
     conda activate "env$PYTHON_VERSION"
     # Install libpng from Anaconda (defaults)
-    conda install ${CONDA_CHANNEL_FLAGS} -c conda-forge libpng "jpeg<=9b" -y
+    conda install ${CONDA_CHANNEL_FLAGS} libpng "jpeg<=9b" -y
```

Comment:
I think you meant to remove CONDA_CHANNEL_FLAGS  here?

Reply:
I think it might still be valuable to have the mechanism to insert channels into these easily.

For example I think we might go through this same exercise when attempting to add Python 3.10 support later on this year.